automation/project/Dependencies.scala

```diff
@@ -7,17 +7,19 @@ object Dependencies {
   val akkaV = "2.6.15"
   val akkaHttpV = "10.1.0"
 
-  val workbenchModelV  = "0.15-808590d"
+  val workbenchModelV  = "0.15-f9f0d4c"
   val workbenchModel: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-model" % workbenchModelV
   val excludeWorkbenchModel = ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = "workbench-model_" + scalaV)
 
-  val workbenchGoogleV = "0.21-808590d"
+  val workbenchGoogleV = "0.21-ae11b9f"
   val workbenchGoogle: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-google" % workbenchGoogleV excludeAll excludeWorkbenchModel
   val excludeWorkbenchGoogle = ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = "workbench-google" + scalaV)
 
-  val workbenchServiceTestV = "0.21-258d483"
+  val workbenchServiceTestV = "0.21-f84f06e"
```

Comment:
updates to the workbench* libs are not strictly necessary, but I was poking around in there and thought it was good to get them up to date.